src/content/configuration/entry-context.md

```diff
@@ -59,6 +59,10 @@ If a string or array of strings is passed, the chunk is named `main`. If an obje
 
 ### Dynamic entry
 
+If a function is passed then it will be invoked on every [make](/api/compiler-hooks/#make) event.
+
+> Note that the `make` event triggers when webpack starts and for every invalidation in watch-mode.
```

Comment:
This is contradicting to other mode option in webpack configuration. Let's reword. Also make event shouldn't be quoted for consistency with first occurrence 
```suggestion
> Note that the make event triggers when webpack starts and for every invalidation when [watching for file changes](/configuration/watch/).
```